packages/core/components/DataTable/DataTable.tsx

```diff
@@ -244,17 +244,17 @@ const DataTable = (props: DataTableProps) => {
         </MediaControls.Section>
       )
     }
+    const getClassNames = (): string => {
+      const isBrushActive = config?.brush?.active && config.legend.position !== 'bottom'
```

Comment:
**Nit:**
Consider organizing the class names in an array and returning the result. This can improve readability and maintainability:

```.tsx
const classes = []
if (logic) classes.push('class-name')
if (!logic) classes.filter(c => c !== 'class-name')
return classes.join(' ')
```

This approach makes the class logic easier to follow, especially when dealing with conditional class assignments.

Reply:
Fixed!